resources/packs/aws/aws_ec2.go

```diff
@@ -716,6 +716,7 @@ func (s *mqlAwsEc2) gatherInstanceInfo(instances []types.Reservation, imdsvVersi
 			}
 
 			args := []interface{}{
+				"platform", core.ToString(instance.PlatformDetails),
```

Comment:
are we sure we want to connect PlatformDetails to `platform`

Reply:
@chris-rock @vjeffrey I have updated field to be `platformDetails` to match the AWS Go SDK

```
 	// The platform details value for the instance. For more information, see AMI
	// billing information fields
	// (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/billing-info-fields.html)
	// in the Amazon EC2 User Guide.
	PlatformDetails *string
```

![image](https://github.com/mondoohq/cnquery/assets/49754039/99c9f76f-f021-4727-869d-4429138eaf67)

Reply:
Please let me know if you would like other changes.